public/_data/repositories/core.yml

```diff
@@ -0,0 +1,39 @@
+section:
+  description: These are the primary repositories for Conjur Open Source and its SDK.
+  categories:
+  - name: Conjur OSS Core
+    description: Conjur OSS Server and Deployment Tools
```

Comment:
Do we use the term `Conjur OSS Server` consistently in the OSS docs and such?  The word `server` feels a little odd to me.  Saying `Conjur OSS Web Service` or something sounds better, maybe?  Just thought.  Feel free to ignore.

Reply:
Oooh, it might actually be inconsistent with this diagram:
<img width="752" alt="Screen Shot 2020-02-07 at 6 09 39 PM" src="https://user-images.githubusercontent.com/26872683/74072510-0a6d0500-49d5-11ea-9dfd-6b9cb3966699.png">

going to take one last look at this to see if I can clarify it